app/models/user.rb

```diff
@@ -19,6 +19,7 @@ class User < ApplicationRecord
 
   has_many :vacancies, dependent: :destroy
   has_many :contacts, dependent: :destroy
+  has_many :notifications
```

Comment:
Did you test this? Witch notification will be? Recipient or sender?

Reply:
@XaoGao you are right, I should have clarified which notifications (sent or received)